centos/stackstorm-compose/docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: '3'
```

Comment:
FYI we keep the Dockerfiles only in this repo.
And so the `docker-compose` deployment resides in https://github.com/stackStorm/st2-docker and K8s deployment in: https://github.com/stackStorm/stackstorm-ha

But for the feedback/demo purposes this PR includes all the deployment configuration as well from my understanding.

Reply:
Agreed; if this PR makes it beyond an experiment, I can work on separating out the other bits.